src/features/emails/components/EmailEditor/EmailEditorFrontend.tsx

```diff
@@ -75,7 +75,9 @@ const EmailEditorFrontend: FC<EmailEditorFrontendProps> = ({
       tools: {
         button: {
           class: Button as unknown as ToolConstructable,
-          config: frame.blockAttributes ? frame.blockAttributes['button'] : {},
+          config: frame.block_attributes
+            ? frame.block_attributes['button']
+            : {},
```

Comment:
This is picking at details, but this could also use the pattern of `frame?.block_attirbutes?['button] ?? {}` that you refactored similar code to in the other pr.

Reply:
You're absolutely right! I'm also not sure why I decided to go with `['button']` instead of just `.button`. I will make this change!

Reply:
Oh I realize I didn't write the original code here, so I guess I couldn't know the motivation. But I will make the change either way! 😊